docs/features/offloading.md

```diff
@@ -1,10 +1,26 @@
 # Offloading
 
-**Workload offloading** is enabled by the **virtual node** abstraction.
-A virtual node is spawned at the end of the peering process in the local (i.e., consumer) cluster, and represents (and aggregates) the subset of resources shared by the remote cluster.
+**Workload offloading** is enabled by a **virtual node**, which is spawned in the local (i.e., consumer) cluster at the end of the peering process, and represents (and aggregates) the subset of resources shared by the remote cluster.
+
 This solution enables the **transparent extension** of the local cluster, with the new node (and its capabilities) seamlessly taken into account by the vanilla Kubernetes scheduler when selecting the best place for the workloads execution.
 At the same time, this approach is fully compliant with the **standard Kubernetes APIs**, hence allowing to interact with and inspect offloaded pods just as if they were executed locally.
 
+## Assigned resources
+
+By default, the virtual node is assigned with 90% of the resources available in the remote cluster. For example:
+* If the remote cluster has 100 vCPUs available, the virtual node will feature 90 vCPU.
+* If now the remote cluster starts some applications that consume 50 vCPUs (i.e., running pods _requesting_ resources), the virtual node is resized to 45 vCPUs (i.e., 90% of (100-50)).
+* If the remote cluster has some autoscaling mechanism that, at some point, double its size and reaches 200 vCPUs (all of them unused by any pod), the virtual node will advertise 180 vCPUs.
+
+This mechanism applies to all the physical resources available in the remote cluster, e.g., CPUs, RAM, GPUs.
+This value can be customized also at run-time using the `--sharing-percentage` option, as documented in the proper [installation section](https://docs.liqo.io/en/v0.7.2/installation/install.html#control-plane).
+
+**Warning 1**: pay attention to _math rounding_. For instance, if your remote cluster has 1 GPU, 0.9GPUs are assigned to the virtual node. Since numbers must be integers, you may end up with a virtual node with _zero_ GPUs.
```

Comment:
```suggestion
**Warning 1**: pay attention to _math rounding_. For instance, if your remote cluster has 1 GPU, with default parameters 0.9 GPUs are assigned to the virtual node. Since numbers must be integers, you may end up with a virtual node with _zero_ GPUs.
```